### PR TITLE
Second Version

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Removed unnecessary scripts from action-scheduler repo and updated other relevant files e.g. dockerfile/pom etc. 

# What has changed
Removal of scripts from main and test folders whilst ensuring acceptance tests still pass

# How to test?
Clone repo and run acceptance tests 

# Links
https://trello.com/c/QzSzSbP6

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[2021 PERFORMANCE: Action worker processor (8)](https://trello.com/c/QzSzSbP6/271-2021-performance-action-worker-processor-8)

**Review by AdamHawtin — APPROVED** [review]

**Review by ryangrundy7 — COMMENTED** [review]
You can also remove `SkippedMessage`, `Peek`, `ExceptionReportResponse` and `ExceptionReport` from the `model/dto` directory. It's not using the Exception manager so doesn't need these

**Review by gavinleeedwards — COMMENTED** [review]
The `README` still refers to action-scheduler.  Would it be better to just have a shell of a README there (just the heading and little else) then for it to be updated when the other part of the task is completed?  Not gonna block the PR on it though so up to you.

**Review by ryangrundy7 — COMMENTED** [review]
You can also get rid of the `exceptionmanager` config in application.yml

File: README.md [hunk 1]
```diff
@@ -1,28 +1,12 @@
-# census-rm-action-worker-processor
+# census-rm-action-worker
 
 [![Build Status](https://travis-ci.com/ONSdigital/census-rm-action-scheduler.svg?branch=master)](https://travis-ci.com/ONSdigital/census-rm-action-scheduler)
```
Comment (ryangrundy7) [inline README.md]: this is still pointing to Action Scheduler, may want to just remove it for now

**Review by gavinleeedwards — APPROVED** [review]
👍

**Pull request merged**